crates/aptos/src/common/utils.rs

```diff
@@ -355,7 +355,7 @@ pub async fn fund_account(
         .body("{}")
         .send()
         .await
-        .map_err(|err| CliError::ApiError(err.to_string()))?;
+        .map_err(|err| CliError::ApiError(format!("Failed to query faucet: {:#}", err)))?;
```

Comment:
I'd change this message to "Failed to fund account"

That message is more friendly to non-technical users.  The error message should end up saying something similar to that anyways I think (I may be wrong)

Reply:
I think mentioning the existence of the faucet is still handy, since it'll remind them they need a faucet (e.g. `--with-faucet`). I'll change it to `Failed to fund account with faucet`